lib/team_data_filterer.rb

```diff
@@ -3,30 +3,6 @@
 
 module TeamDataFilterer
```

Comment:
Without the need for the team api (and the filtering of the data), should this be called something else now?